lib/utils/sendNotifications.js

```diff
@@ -0,0 +1,27 @@
+const users = require('../../lib/modelManagers/followModel');
+const sendVerficationMail = require('../../lib/utils/emailService/emailNotification');
+const articleModelManager = require('../../lib/modelManagers/articlemodel');
+const user = require('../../lib/modelManagers/userProfileModel');
+
+const getUsersToNotify = async (id, title, article) => {
+  let UsersToNotify = await users.getUserFollowees(id);
+  const author = await articleModelManager.getArticlesby('authorId', id, 2, 0);
+  const authorName = `${author[0].dataValues.articleAuthor.dataValues.firstname} ${author[0].dataValues.articleAuthor.dataValues.lastname}`;
```

Comment:
Please what is the `2` and  `0` you are passing into the `getArticlesby` function

Reply:
I was trying to pass limit and offset int the function but it is not necessary, so I have fixed it.